scripts/wrapper/apt-get.py

```diff
@@ -7,7 +7,11 @@
 
 def main(argv=sys.argv[1:]):
     max_tries = 10
-    known_error_strings = ['Failed to fetch', 'Hash Sum mismatch']
+    known_error_strings = [
+        'Failed to fetch',
```

Comment:
We use 
`RUN locale-gen en_US.UTF-8`
and
`ENV LANG en_US.UTF-8`

LC_ALL=C might be a better solution. https://unix.stackexchange.com/questions/87745/what-does-lc-all-c-do#